chapters/operatorsandexpressions.tex

```diff
@@ -531,10 +531,10 @@ \subsection{Numeric Functions and Conversion Functions}\label{numeric-functions-
   \lstinline!(if leftJustified then "-" else "") + String(minimumLength) + "d"!
 \end{itemize}
 
-Form of the \lstinline!format! string:
-According to ANSI-C the format string specifies one conversion specifier (excluding the leading \%), shall not contain length modifiers, and shall not use `\lstinline!*!' for width and/or precision.
-For all numeric values the format specifiers `\lstinline!f!', `\lstinline!e!', `\lstinline!E!', `\lstinline!g!', `\lstinline!G!' are allowed.
-For integral values it is also allowed to use the `\lstinline!d!', `\lstinline!i!', `\lstinline!o!', `\lstinline!x!', `\lstinline!X!', `\lstinline!u!', and `\lstinline!c!' format specifiers (for non-integral values a tool may round, truncate or use a different format if the integer conversion characters are used).
+The ANSI-C style \lstinline!format! string (which cannot be combined with any of the other named arguments) consists of a single conversion specifier without the leading \%.
+It shall not contain a length field specifier, and shall not use `\lstinline!*!' for width and/or precision.
+For all numeric values the type field specifiers `\lstinline!f!', `\lstinline!e!', `\lstinline!E!', `\lstinline!g!', `\lstinline!G!' are allowed.
+For integral values it is also allowed to use the `\lstinline!d!', `\lstinline!i!', `\lstinline!o!', `\lstinline!x!', `\lstinline!X!', `\lstinline!u!', and `\lstinline!c!' type field specifiers (for non-integral values a tool may round, truncate or use a different format if the integer conversion characters are used).
```

Comment:
```suggestion
For all numeric values the conversion specifiers `\lstinline!f!', `\lstinline!e!', `\lstinline!E!', `\lstinline!g!', `\lstinline!G!' are allowed.
For integral values it is also allowed to use the `\lstinline!d!', `\lstinline!i!', `\lstinline!o!', `\lstinline!x!', `\lstinline!X!', `\lstinline!u!', and `\lstinline!c!' conversion specifiers (for non-integral values a tool may round, truncate or use a different format if the integer conversion specifiers are used).
```
That character is the "conversion specifier".

Reply:
Before applying this change, I think we should just take a look at some relevant modern references for the terminology.  Also, _type field specifier_ has the advantage of being more distinct from _conversion specification_.

References with terminology similar to _type field specifier_:
- https://en.wikipedia.org/wiki/Printf_format_string "Type field"
- [Microsoft](https://learn.microsoft.com/en-us/cpp/c-runtime-library/format-specification-syntax-printf-and-wprintf-functions?view=msvc-170#type-field-characters) "Type field characters"
- [IBM](https://www.ibm.com/docs/en/i/7.2?topic=functions-printf-print-formatted-characters) "type character"

References with terminology similar to _conversion specifier_:
- [N1570](https://port70.net/~nsz/c/c11/n1570.html#7.21.6) "conversion specifier"

Other:
- [cplusplus.com](https://cplusplus.com/reference/cstdio/printf/) "specifier character" (and uses _format specifier_ for what we currently refer to as _format specification_)

Feel free to mention more references, and I'll add them by updating this comment.  So far, however, it looks to me as if _type field specifier_ is the winner even though it isn't the original terminology.

Reply:
Well, the Wikipedia article is for multiple languages, and the Microsoft article is all over the place and also calls it "type conversion specifier character", and discusses "floating-point conversion specifier" (a, A, e, E, f, F, g, G).

And https://en.cppreference.com/w/c/io/fprintf calls it conversion specifier.
Thus I prefer to use the name in the standard.